.github/workflows/check-template-and-add-labels.yml

```diff
@@ -4,7 +4,7 @@ on:
   issues:
     types: [opened, edited, labeled, unlabeled, reopened]
   pull_request_target:
-    types: [opened, edited, labeled, unlabeled, reopened]
+    types: [opened, edited, labeled, unlabeled, reopened, synchronize]
```

Comment:
hi there 👋 I think this will fix the issue. Because of this reasons:

> The synchronize event is triggered when new commits are added to the branch of a pull request.
> The edited event is triggered when metadata of the pull request itself is updated, NOT when commits are added.

From what I've seen, the issue is always repro for me, when I add a new commit in a PR (without changing anything else from the PR). So this makes sense.
I don't think it was a rate limit issue, rather the behaviour stated above

Reply:
Ok great!